### ISSUE TITLE
Tooltip for changing overlay selection? 

### ISSUE THREAD

**Issue by canuckjacq** [HackForEquality/YesEquality#5]
I love the tap to change, but I think it might need something to explain it?

**Comment by Lmd64-zz** [comment]
Yep, discoverability's an issue there. Also, you can drag the overlay into a different corner too before and after you take the photo (another undiscoverable feature!)

**Comment by canuckjacq** [comment]
If we could do something that drew attention to it and said "tap me!" Honestly, I think people will discover the dragging. I think after Twibbons, dragging is expected, but the tap to change is unusual (but elegant!)

**Comment by adam-govan** [comment]
you could show a little label at the bottom of the image indicating that it can be played with.. I'll leave the wording to you
